tests/defer/src/commonTest/kotlin/test/DeferNormalizedCacheTest.kt

```diff
@@ -370,7 +374,10 @@ class DeferNormalizedCacheTest {
                     emit(networkResponse as ApolloResponse<D>)
                   }
                   delay(10)
-                  throw ApolloNetworkException("Network error")
+                  emit(ApolloResponse.Builder(requestUuid = uuid, operation = query, data = null)
```

Comment:
Can you add a comment that `NetworkTransport.execute` must not throw? (which I believe to be the new contract?)